tests/foreman/ui/test_remoteexecution.py

```diff
@@ -0,0 +1,227 @@
+"""Test class for Remote Execution Management UI"""
+from robottelo.decorators import stubbed, tier1, tier2
+from robottelo.test import UITestCase
+
+
+class TestRemoteExecution(UITestCase):
+    """Test class for remote execution feature"""
+
+    @stubbed()
+    @tier1
+    def test_positive_create_simple_job_template(self):
+        """Create a simple Job Template
+
+        @Feature: Remote Execution
+
+        @Steps:
+
+        1. Navigate to Hosts -> Job Templates
+        2. Enter a valid name
+        3. Populate the template code
+        4. Navigate to the job tab
+        5. Enter a job name
+        6. Click submit
+
+        @Assert: The job template was successfully created
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier1
+    def test_positive_create_job_input_template(self):
+        """Create a Job Template using input
+
+        @Feature: Remote Execution
+
+        @Steps:
+
+        1. Navigate to Hosts -> Job Templates
+        2. Enter a name
+        3. Navigate to the job tab
+        4. Enter a job name
+        5. Click the +Add Input button
+        6. Add an appropriate name
+        7. Choose an input type
+        8. Populate the template code and reference the newly created input
+        9. Click submit
+
+        @Assert: The job template was successfully created
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier1
+    def test_negative_create_job_template(self):
+        """Create Job Template with duplicate name
+
+        @Feature: Remote Execution
+
+        @Steps:
+
+        1. Create a valid job template
+        2. Enter a name that has already been used
```

Comment:
You might want to add a step of pre-creating the 1st job template first. You can use the @Setup section of the docstring for such steps